client/dashboard/plugins/plugin/style.scss

```
@@ -0,0 +1,14 @@
+.plugin-icon-placeholder {
+	width: 40px;
+	height: 40px;
+	background-color: #c3c4c7;
```

Should we use some variable here? Like:

```suggestion
	background-color: var(--color-neutral-10, #c3c4c7);
```

---

Good catch, thanks!